javascript/javascript1/week1/ez-namey/ez-namey.js

```diff
@@ -0,0 +1,36 @@
+
+const firstWords = [
+    "Dynamic",
+    "Transparent",
+    "Inovative",
+    "Sustainable",
+    "Customer-Centric",
+    "Collaborative",
+    "Adaptable",
+    "Data-Driven",
+    "Empatthetic",
+    "Scalable"
+];
+const secondWords = [
+    "Evolving",
+    "Balancing Positivity & Realism",
+    "Ethical",
+    "Empowering",
+    "Tech-savvy",
+    "Global",
+    "Resilient",
+    "User-Friendly",
+    "Secure",
+    "Reliable",
+    "Fast-Paced"
+];
+
+const startupName = "NouGol"; // First letters of my first and last name ;)
+
+
+const randomFirst = Math.floor(Math.random() * 10);
+const randomSecond = Math.floor(Math.random() * 10);
```

Comment:
Here the variables `randomFirst` and `randomSecond` should be placed before `startupName` and then using the randomly generated numbers between 0-10 to pick a word from the arrays you created.

The reason why you have to put them before the `startupName` variable is because in Javascript you cannot call a variable unless it has been delcared before you want to call it (from top to bottom).

Hint:
`const startupName = first words array [random number between 0-10] + " " + second words array [random number between 0-10]`